script.js

```diff
@@ -1,4 +1,149 @@
+// First time main function runs, playerOneMode runs
+// {both dice results are saved into an array (1st input - click to roll dice)
+// user inputs either A or B to choose first or second dice (2nd input - select dice)
+// output returns combined result(this is done by forcing them into strings using the googled method) - Once this condition is fulfilled, result is saved in a second array, and Global Variables are reset, together with first array collection.
+// Repeat steps 1 to 4, and compare the results
+
+//Googled Method
+//______________
+// var a = 5;
+// var b = 6;
+// console.log("" + a + b);
+
+// OR
+
+// const numbersAsString = `${5}${6}`;
+
+//______________
+var playerOneRolls = [];
+var playerTwoRolls = [];
+var comparisonArray = [];
+//______________
+var playerOneMode = 1;
```

Comment:
nice work here. But you might want to consider using a different data type for switching mode? (Booleans) 👍